core-bundle/src/Resources/contao/themes/flexible/main.css

```diff
@@ -2175,10 +2179,14 @@ fieldset.collapsed > legend {
 		transform:translateY(-40px);
 	}
 	#header h1 a {
-		width:22px;
+		min-width:22px;
```

Comment:
I think `width:22px` was only needed in combination with `overflow:hidden` and can be left out entirely. Can you check this please?

Reply:
Yes, it is still necessary because with out the badge-title we have no width for the container with background icon:
![image](https://user-images.githubusercontent.com/14016098/104003660-ff9d9000-51a2-11eb-982c-ebef5c317944.png)